### ISSUE TITLE
re-creating server using docker compose wipes save game and backups

### ISSUE THREAD

**Issue by GsakuL** [wolveix/satisfactory-server#285]
**Describe the Bug**
Yesterday, we started playing with `1.0`. After reading up on #260 I've decided to use `network_mode: host` for now.
Today, I saw the new version and `pull`ed. I removed `network_mode` and adjusted the ports (so they are non-default).
The container got re-created, but it failed to download game files
```
SteamUpdater: Error: Steam needs to be online to update.  Please confirm your network connection and try again.
```
I wasn't sure, so I re-enabled `network_mode`, and it worked. However, upon the SF server being reachable again, I noticed, that it was "unclaimed" and no savegame could be loaded.

Sure enough, the save and backup directories where empty.

Because the Path is mapped, I did not do a manual backup/copy since i assumed, these directories where never `rm`'d

**Your Runtime Command or Docker Compose File**
Please censor anything sensitive. Your runtime command might look like this:
(Port numbers are censored. Mappings are ignored due to `network_mode: host`)
```yaml
services:
    satisfactory:
        container_name: 'satisfactory'
        hostname: 'satisfactory-server'
        image: 'wolveix/satisfactory-server:latest'
        ports:
            - '12345:12345/udp'
            - '12345:12345/tcp'
        volumes:
            - './config:/config'
        environment:
            MAXPLAYERS: 4
            PGID: 1000
            PUID: 1000
            ROOTLESS: false
            STEAMBETA: false
            SERVERGAMEPORT: 12345
        restart: unless-stopped
        network_mode: 'host'
```

**System Specs (please complete the following information):**
If you're on Linux, just paste the following block as a single command, and paste the output here.

```txt
===== START ISSUE REPORT =====
OS:  Linux MyServer 6.6.50-1-lts #1 SMP PREEMPT_DYNAMIC Sun, 08 Sep 2024 12:27:02 +0000 x86_64 GNU/Linux
CPU: AMD EPYC 7282 16-Core Processor
RAM: 14GB/15GB
HDD: 147GB/388GB (40% used)
===== END ISSUE REPORT =====

```

**Additional Context**
(none)

**Comment by wolveix** [comment]
Hey @GsakuL! Really sorry to hear about this, this was caused by a bug that's since been fixed (#266).

The Steam issue you encountered would have been a rate limit or network issue, they happen intermittently. Please make sure to pull the latest version of the Docker image, and test with a new save to ensure that things are working as expected.

**Comment by GsakuL** [comment]
oh I'm sorry. I did not search good enough, and didn't see the other issue.

ok. I'd expect a different message for a rate limit, but that's a steam thing. Good to know.

since i just pulled, `latest` is `v1.7.3`/`b02914a46b1d`.

I did a `docker restart satisfactory` and a `docker compose up -d --force-recreate` (only SF container), and both times the server came up with the new save as expected.

"luckly" it was "just" ~6.5h of progress.

Still tho, great work! :+1:
